.github/workflows/Release.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish a release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+      - name: Publish the artifacts
+        env:
+          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME }}
+          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD }}
+          # TODO(egorand): Figure out if the publishing plugin needs GPG keys and how they get fed
```

Comment:
You cannot publish without the artifacts being signed

Reply:
Yeah makes sense. I'll add the keys.